# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../kvirc/kvirc-gcc4.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
diff -Nur kvirc-3.2.0.orig/src/kvirc/kernel/kvi_irccontext.h kvirc-3.2.0/src/kvirc/kernel/kvi_irccontext.h
--- kvirc-3.2.0.orig/src/kvirc/kernel/kvi_irccontext.h	2004-11-22 04:13:17.000000000 +0200
+++ kvirc-3.2.0/src/kvirc/kernel/kvi_irccontext.h	2005-09-27 23:19:08.000000000 +0300
@@ -29,6 +29,7 @@
 #include <qobject.h>
 
 class KviChannel;
+class KviConsole;
 class KviQuery;
 class KviQuery;
 class KviFrame;
diff -Nur kvirc-3.2.0.orig/src/kvirc/module/kvi_moduleextension.h kvirc-3.2.0/src/kvirc/module/kvi_moduleextension.h
--- kvirc-3.2.0.orig/src/kvirc/module/kvi_moduleextension.h	2004-09-10 00:51:07.000000000 +0300
+++ kvirc-3.2.0/src/kvirc/module/kvi_moduleextension.h	2005-09-27 23:19:08.000000000 +0300
@@ -95,6 +95,8 @@
 
 typedef KviPtrList<KviModuleExtensionDescriptor> KviModuleExtensionDescriptorList;
 
+class KviModuleExtensionManager;
+
 extern KVIRC_API KviModuleExtensionManager * g_pModuleExtensionManager;
 
 class KVIRC_API KviModuleExtensionManager
diff -Nur kvirc-3.2.0.orig/src/kvirc/ui/kvi_console.h kvirc-3.2.0/src/kvirc/ui/kvi_console.h
--- kvirc-3.2.0.orig/src/kvirc/ui/kvi_console.h	2005-02-25 19:57:45.000000000 +0200
+++ kvirc-3.2.0/src/kvirc/ui/kvi_console.h	2005-09-27 23:19:08.000000000 +0300
@@ -46,6 +46,8 @@
 class KviIrcServer;
 class KviIrcNetwork;
 class KviProxy;
+class KviIrcSocketMonitor;
+class KviIrcServerDataBase;
 #ifndef COMPILE_ON_WINDOWS
 	class KviChannel;
 	class KviQuery;
